vm/src/obj/objstr.rs

```diff
@@ -1590,6 +1616,36 @@ fn perform_format(
     Ok(vm.ctx.new_str(final_string))
 }
 
+fn perform_format_map(
+    vm: &VirtualMachine,
+    format_string: &FormatString,
+    dict: &PyObjectRef,
+) -> PyResult {
+    let mut final_string = String::new();
+    for part in &format_string.format_parts {
+        let result_string: String = match part {
+            FormatPart::AutoSpec(_) => {
+                return Err(
+                    vm.new_value_error("Format string contains positional fields".to_string())
+                );
+            }
```

Comment:
You can merge this match arm and the previous one by using the `|` character: `FormatPart::AutoSpec(_) | FormatPart::IndexSpec (_, _) => { ... }`

Reply:
:+1: